app/src/main/java/io/intrepid/contest/screens/contestcreation/addcategoriestocontest/AddCategoriesFragment.java

```diff
@@ -0,0 +1,65 @@
+package io.intrepid.contest.screens.contestcreation.addcategoriestocontest;
+
+import android.content.Context;
+import android.os.Bundle;
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+import android.widget.EditText;
+import android.widget.Toast;
+
+import java.util.List;
+
+import butterknife.BindView;
+import butterknife.OnClick;
+import io.intrepid.contest.R;
+import io.intrepid.contest.base.BaseFragment;
+import io.intrepid.contest.base.PresenterConfiguration;
+import io.intrepid.contest.models.Category;
+import io.intrepid.contest.models.Contest;
+import io.intrepid.contest.screens.contestcreation.ContestActivityContract;
+import io.intrepid.contest.screens.contestcreation.ContestCreationFragment;
+import io.intrepid.contest.screens.contestcreation.ValidatableView;
```

Comment:
Double check that you've optimized imports, there are a lot of unused ones here.